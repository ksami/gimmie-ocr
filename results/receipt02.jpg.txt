Actual | Predicted | OCR Conf
HAH-EZFGE F81 P9700 C9 | (CASH, 45) | 54.016052
TAMPINES HALL | (TAMPINES MALL, 92) | 65.909721
Q T399199S CGMFGI 5 | (SENGCHOON EGG 55 G, 36) | 64.742020
.31-12 S1W3DON 529510 | (RAD SHRED-MOZZ 250G, 41) | 62.308903
TEL. 57861953 | (TOTAL, 36) | 66.269867
RES N0. RCS 1.1.191.1 | (REG, 60) | 75.439323
GST N0. M4-000457S-0 | (GST, 90) | 66.932068
SGD | (SGD, 100) | 73.162628
YOPLT L/YOG BKB 1256 1.20 | (YOPLAIT YOGURT 1, 86) | 75.343384
VOPLT L/YOG MNG 1250 1.20 | (YOPLAIT YOGURT 1, 86) | 75.025864
YOPLAIT YOGURT 1 0.45- | (YOPLAIT YOGURT 1, 95) | 72.340515
SUBTCTAL 1.95 | (SUBTOTAL, 79) | 75.314163
GROUP DISCOUNTABLE 0.00 | (GROUP DISCOUNTABLE, 95) | 77.104782
SUBTCTA.. 1.95 | (SUBTOTAL, 68) | 73.496246
TOIAL IDCIUMS GST 1.95 | (GST, 90) | 67.115997
CRSH 2.00 | (CASH, 68) | 71.079903
CHANGE DUE 0.05 | (CHANGE DUE, 90) | 76.951813
III-III ITEHS PURCHASED. 2 0 | (ITEMS PURCHASED, 86) | 62.731201
TOTAL SAVINGS1 0.45 | (TOTAL, 90) | 75.402344
N0. 5ALE ITEMS. 2 | (NO. SALE ITEMS, 77) | 71.980026
TMHR YOU F09 89099199 5019 US 1 | (YOPLAIT YOGURT 1, 86) | 63.553200
. K969 FGCEHTI 99F 99111991 M0119 15 03915. | (RECEIPT NO., 36) | 48.325863
599 NWM 901199 905299 M-SXOPG. | (NSOY FR S/MILK A, 38) | 55.122002
SHM MEI | (NSOY FR S/MILK A, 51) | 77.005997
. RECEIPT N0. . 46984 | (RECEIPT NO., 86) | 77.045578
.9 J S01TME4R999 09.3237 19.46984 | (NO. SALE ITEMS, 32) | 31.447144
M... 9- 30/12/08 A | (BASIL 30G, 40) | 41.661156
