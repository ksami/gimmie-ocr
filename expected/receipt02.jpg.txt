FAIRPRICE XTRA
TAMPINES MALL
!4 Tampines Central 5
!#B1-12 Singapore 529510
!TEL: 67861953
!REG No: RCS 1.1.191.1
!GST No: M4-0004578-0
SGD
YOPLT L/YOG B/B 125G 1.20
YOPLT L/YOG MNG 125G 1.20
YOPLAIT YOGURT 1 0.45-
SUBTOTAL 1.95
GROUP DISCOUNTABLE 0.00
SUBTOTAL 1.95
TOTAL Includes GST 1.95
CASH 2.00
CHANGE DUE 0.05
ITEMS PURCHASED: 2
TOTAL SAVINGS: 0.45
NO. SALE ITEMS: 2
